ui/src/pages/frontsite/StaticPage/StaticPage.js

```diff
@@ -0,0 +1,50 @@
+import { Error } from '@components';
+import {
+  ILSHeaderPlaceholder,
+  ILSParagraphPlaceholder,
+} from '@components/ILSPlaceholder';
+import { getStaticPageByRoute } from '@config/uiConfig';
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Container, Header } from 'semantic-ui-react';
+import ReactHtmlParser from 'react-html-parser';
+import _isEmpty from 'lodash/isEmpty';
+
+export default class StaticPage extends Component {
+  componentDidMount() {
+    const staticPage = getStaticPageByRoute(this.props.match.path);
+    const staticPageID = staticPage['apiURL'];
+    this.props.fetchStaticPageDetails(staticPageID);
+  }
+
+  parseStaticPageContent = () => {
+    const { data } = this.props;
+    if (!_isEmpty(data)) {
+      // unescapes HTML and strips unsafe tags
+      const parsed = new DOMParser().parseFromString(data.content, 'text/html');
```

Comment:
please verify this, I found it as a method which does not allow for XSS

Reply:
are you sure we need all this? if the `data.content` contains some HTML, it should just render when you add it to the DOM. What it is happening here?